docs/app/data/components-page.json

```diff
@@ -1229,8 +1229,14 @@
                 },
                 {
                     "title": "Color Service",
+                    "component": "ComponentsColorServiceComponent",
+                    "version": "Angular"
+                },
```

Comment:
I forgot to mention one documentation feature that we have added for Angular 4. Could you add a "usage" property here, listing the module required for the color service?

````javascript
                        {
                            "title": "Module",
                            "content": "ColorServiceModule"
                        }
````